client/components/overviewWidget/AddToCart.jsx

```diff
@@ -39,27 +41,69 @@ class AddToCart extends React.Component {
           console.error(err);
         });
     }
+
+    if (this.props.selectedStyle !== prevProps.selectedStyle) {
+      const newSelectedStyleData = this.state.styles.filter((style) => {
+        return style.style_id === parseInt(this.props.selectedStyle);
+      });
+      let skus = Object.values(newSelectedStyleData[0].skus);
+      let sizes = [];
+      skus.forEach((sku) => {
+        sizes.push(sku.size);
```

Comment:
I noticed that, for this product, and the 'Forrest Green' style, the API provided the "XL" size twice. Here would be a good place to enforce uniqueness on this set, among other ways of handling duplicate sizes.